board_rules.md

```diff
@@ -0,0 +1,59 @@
+# Board rules 
+
+The description of the team workflow for the board.
+
+## Description
+
+The board consists of 5 columns:
+
+- `Common backlog`- includes all tasks that should be done.
+- `Sprint backlog` - includes the list of tasks for the current sprint.
+- `In Progress` - the list of tasks that the team works on.
+- `In Review` - the list of tasks that are under review process.
+- `Done` - the list of tasks that are considered as completed. If a task is no longer makes sense, then it must be closed with an appropriate comment after agreement with the team.
+
+### Usage
+
+The issue moves from left to right and can't skip any step (board section). The exception is when the team decides that the task cannot be completed or it needs to be canceled (read the description of **Done** section).
+The main flow is:
+ `
+Common backlog -> Sprint backlog -> In progress -> In review -> Done.
+`
+
+### Common backlog
+
+Every issue is placed in a common backlog when it is created.
+The issue must contain a clear title and description with definition of done (**DoD**).
+Select necessary labels, if applicable. For example, if you add a task to create documentation, put the label "Documentation" ([label's description](https://github.com/ios-course/ironfoudation-team-project/labels)). 
+Don't forget to choose corresponding project in the `Projects` section on pages, describing PR's and issue's contain.
+
+### Sprint backlog
+
+Before sprint planning the issues place into Sprint backlog by PO.
+A milestone should be put to the issue at the moment of moving it into the Sprint backlog.
+During the sprint planning process, the issues from the sprint backlog are estimated (labels with estimation are assigned).
+
+**Estimation** is carried out with the participation of all team members online using the **Scrum Poker** technology on the resource [planningpokeronline](https://planningpokeronline.com).
+
+Except estimation labels in project also provided [labels](https://github.com/ios-course/ironfoudation-team-project/labels). 
```

Comment:
nit: we had the same link above already. Do we need this paragraph?

Reply:
Yes, you are right. This is duplicate information. 
The paragraph is removed.